pkgs/tools/misc/screamingfrogseospider/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, lib, fetchurl, dpkg, oraclejre }:
+with lib;
+let
+  version = "11.3";
+  url = "https://download.screamingfrog.co.uk/products/seo-spider/screamingfrogseospider_${version}_all.deb";
+  sha256 = "a6a7b9434dadcd0ca5d486df16386ce317ed7d4d2f7acc2730156e13859368f9";
+in stdenv.mkDerivation rec {
```

Comment:
Why did you felt like you need the `let` + `in` syntax? Why not just use everything inside `mkDerivation`?

Reply:
Honestly not sure, will push an update later to rectify this. Thanks for the feedback.